standalone-examples/build.gradle

```diff
@@ -131,6 +131,22 @@ task startNoopReader(type: JavaExec) {
     }
 }
 
+task scriptStreamCutsCli(type: CreateStartScripts) {
```

Comment:
Why are we creating a separate CLI for stream cuts rather than have a single CLI that encompasses all features?

Reply:
I followed the pattern of existing ConsoleRW and StateSynchronizer applications, which have their own CLI each. Indeed, we could consider to do a refactor of standalone samples and create a unified CLI for all applications, but in this case the changes in code and documentation may be important enough to be a separate PR.